guides/nuxtjs-starter-guide/configuring.mdx

```diff
@@ -0,0 +1,102 @@
+This means that for Nuxt sites, CloudCannon out of the box only has the source editor available.
+
+#### Interlude: Editing terminology
+
+CloudCannon has a few concepts that make organizing data easier:
+
+* **Data**\: These are standalone data files. This is ideal for site configuration or files that don't need to repeat. Data is edited by altering a single file.
+* **Collections**\: A folder of files that represent a repeatable data format. This is a good fit for pages, blogs, staff members, recipes, etc. Collections are edited by altering any file or adding more files.
+
+Using the [Nuxt Content](https://content.nuxtjs.org/) module makes integrating your Nuxt site with CloudCannon easy.
+This module has clear anologues to the organization of data mentioned above.
+Most importabtly, it requires that you separate your site content from the templating and rendering logic.
+This makes it easiest to integrate the site into the CMS.
+
+When using the Nuxt Content module, we recommend grouping related content in directories.
+For instance, if your site has a blog with multiple posts, it's a good idea to keep those posts in a `posts/` folder.
+
+<comp.Notice info_type="info">
+  CloudCannon does not support Server-side Rendering as our build and hosting environments are decoupled.
+</comp.Notice>
+
+CloudCannon supports a set of file formats for files in Data or Collections:
+
+* Structured data files: JSON, YAML, TOML, CSV, TSV
+* Markup files: HTML, Markdown
+* Combination files: HTML with front matter, Markdown with front matter
+
+Front matter refers to a section at the top of a markup file that contains structured data. CloudCannon supports JSON, YAML and TOML front matter. Markdown comes in many flavours, so be sure to checkout configuring your Markdown engine.
+
+*<u><strong>If you need another supported format, please contact support and we would be happy to look into it.</strong></u>*
+
+### 3: Updating Nuxt to use file data
+
+Now that we have the terminology it’s time to organize our content.
+For Document-driven sites, in your Nuxt page component(s) you can fetch your page data like so:
+
+<comp.CodeBlock language="javascript" source="pages/[...slug].vue">
+```
+  const { page } = useContent();
+```
+</comp.CodeBlock>
+This catch-all route will fetch the relevant file in your `content/` directory, based on the URL you have navigated to.
+For example, navigating to `/about/` will fetch the `content/about.md`, if it exists.
+
+If you are not using Document-driven mode, you can instead fetch your page data using the [queryContent composable](https://content.nuxtjs.org/api/composables/query-content), like so:
+
+<comp.CodeBlock language="javascript" source="pages/about.vue">
+```
+	const page = await queryContent().where({ _path: '/about' }).findOne();
+```
+</comp.CodeBlock>
+
+CloudCannon edits content stored in the local file system. This means we need to change the fetch to use an `fs` call instead. For example:
```

Comment:
`For example:` not followed by an example

Reply:
Example left as an exercise for the reader 😤